app/src/main/java/cos/premy/mines/data/MineCoord.java

```diff
@@ -0,0 +1,15 @@
+package cos.premy.mines.data;
+
+/**
+ * Created my Tim Myers on 02/03/21
+ */
+public class MineCoord {
+    public int x, y, z;
+
+    public MineCoord() {}
+    public MineCoord(int x, int y, int z) {
+        this.x = x;
+        this.y = y;
+        this.z = z;
+    }
```

Comment:
This new class was helpful since we have to pass multiple coords back up to the Grid instance